deployment.rb

```diff
@@ -133,6 +134,12 @@ def load_configuration
 
     config['bundler_use_sudo'] = config['ruby_installer'] == 'system'
 
+    # test environment should use precompiled, minified, digested assets like production,
+    # unless it's being used for unit tests. This logic should be kept in sync with
+    # the logic for setting config.assets.* under dashboard/config/.
+    ci_test = !!(ENV['UNIT_TEST'] || ENV['CI'])
+    config['pretty_js'] = [:development, :staging].include?(rack_env) || (rack_env == :test && ci_test)
```

Comment:
yes, this will allow locals.yml override, which happens in `config.merge! local_config ` below